emperor/support_files/js/color-view-controller.js

```diff
@@ -132,16 +132,26 @@ define([
           var data = decompViewDict.setCategory(attributes, scope.setPlottableAttributes, category);
 
           if (scaled) {
-            scope.setSlickGridDataset({});
-            scope.$gridDiv.hide();
+            //build plottables as all non-numerics
+            var nonNumeric = uniqueVals.filter(function(v) { return isNaN(v)});
```

Comment:
I see, but I don't think we should do that in this case.

On (May-09-16| 8:39), Joshua Shorenstein wrote:
> > @@ -132,16 +132,26 @@ define([
> >            var data = decompViewDict.setCategory(attributes, scope.setPlottableAttributes, category);
> >  
> >            if (scaled) {
> > -            scope.setSlickGridDataset({});
> > -            scope.$gridDiv.hide();
> > +            //build plottables as all non-numerics
> > +            var nonNumeric = uniqueVals.filter(function(v) { return isNaN(v)});
> 
> I can, but I wanted to keep this consistent between all the filter functions.
> 
> ---
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/emperor/pull/468/files/c5c2d10589eb2a5c7dd4d326dfa32745e0ee82cc#r62520914